docs/en/ingest-management/troubleshooting.asciidoc

```diff
@@ -178,4 +178,37 @@ If this happens, select **Force unenroll** from the *Actions* menu in the
 Agents list.
 
 This will invalidate all API keys related to the agent and change the status to
-`inactive` so that the agent no longer appears in {fleet}. 
+`inactive` so that the agent no longer appears in {fleet}.
+
+[discrete]
+[[endpoint-unauthorized]]
+== API key is unauthorized to send telemetry to `.logs-endpoint.diagnostic.collection-*` indices
+
+Telemetry is turned on by default in {kib}. This helps us learn about the
```

Comment:
Not sure if this should say "in Kibana" or "in the Elastic Stack.

Reply:
Say Elasticstack

Reply:
*Elastic Stack